julea-test-afl-smd-backend
==25372== Memcheck, a memory error detector
==25372== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==25372== Using Valgrind-3.14.0 and LibVEX; rerun with -h for copyright info
==25372== Command: ./build-gcc-asan/test-afl/julea-test-afl-smd-backend
==25372== 
** (process:25372): DEBUG: 20:03:11.825: ../backend/smd/sqlite.c:207:backend_init: init useing path=/mnt2/julea/smd300
==25372== 
==25372== FILE DESCRIPTORS: 3 open at exit.
==25372== Open file descriptor 2: /src/julea/x
==25372==    <inherited from parent>
==25372== 
==25372== Open file descriptor 1: /src/julea/x
==25372==    <inherited from parent>
==25372== 
==25372== Open file descriptor 0:
==25372==    <inherited from parent>
==25372== 
==25372== 
==25372== HEAP SUMMARY:
==25372==     in use at exit: 46,076 bytes in 367 blocks
==25372==   total heap usage: 1,467 allocs, 1,100 frees, 611,722 bytes allocated
==25372== 
==25372== 4 bytes in 1 blocks are still reachable in loss record 2 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x4911CF2: g_private_impl_new (gthread-posix.c:1007)
==25372==    by 0x491225B: g_private_get_impl (gthread-posix.c:1035)
==25372==    by 0x491225B: g_private_get (gthread-posix.c:1062)
==25372==    by 0x48E9AEC: thread_memory_from_self (gslice.c:507)
==25372==    by 0x48E9AEC: g_slice_alloc (gslice.c:1003)
==25372==    by 0x48BB05D: g_hash_table_new_full (ghash.c:717)
==25372==    by 0x48DCACA: g_quark_init (gquark.c:61)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 4 bytes in 1 blocks are still reachable in loss record 4 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x4911CF2: g_private_impl_new (gthread-posix.c:1007)
==25372==    by 0x4912303: g_private_get_impl (gthread-posix.c:1035)
==25372==    by 0x4912303: g_private_replace (gthread-posix.c:1104)
==25372==    by 0x508C718: g_module_set_error_unduped (gmodule.c:269)
==25372==    by 0x508C718: g_module_set_error (gmodule.c:276)
==25372==    by 0x508C718: g_module_open (gmodule.c:498)
==25372==    by 0x485A46D: j_backend_load (jbackend.c:70)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DA8B: j_init (jcommon.c:171)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 4 bytes in 1 blocks are still reachable in loss record 5 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x4911CF2: g_private_impl_new (gthread-posix.c:1007)
==25372==    by 0x491225B: g_private_get_impl (gthread-posix.c:1035)
==25372==    by 0x491225B: g_private_get (gthread-posix.c:1062)
==25372==    by 0x48D3A83: g_logv (gmessages.c:1353)
==25372==    by 0x48D3D4E: g_log (gmessages.c:1432)
==25372==    by 0x55FD715: backend_init (sqlite.c:207)
==25372==    by 0x485C34E: j_backend_smd_init (jbackend.c:663)
==25372==    by 0x485DBCB: j_init (jcommon.c:191)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 4 bytes in 1 blocks are still reachable in loss record 6 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x4911CF2: g_private_impl_new (gthread-posix.c:1007)
==25372==    by 0x491225B: g_private_get_impl (gthread-posix.c:1035)
==25372==    by 0x491225B: g_private_get (gthread-posix.c:1062)
==25372==    by 0x48D3431: g_log_structured_array (gmessages.c:1956)
==25372==    by 0x48D3431: g_log_structured_array (gmessages.c:1943)
==25372==    by 0x48D390C: g_log_default_handler (gmessages.c:3158)
==25372==    by 0x48D3B5E: g_logv (gmessages.c:1370)
==25372==    by 0x48D3D4E: g_log (gmessages.c:1432)
==25372==    by 0x55FD715: backend_init (sqlite.c:207)
==25372==    by 0x485C34E: j_backend_smd_init (jbackend.c:663)
==25372==    by 0x485DBCB: j_init (jcommon.c:191)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 4 bytes in 1 blocks are still reachable in loss record 7 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x4911CF2: g_private_impl_new (gthread-posix.c:1007)
==25372==    by 0x491225B: g_private_get_impl (gthread-posix.c:1035)
==25372==    by 0x491225B: g_private_get (gthread-posix.c:1062)
==25372==    by 0x48A7018: g_get_charset (gcharset.c:185)
==25372==    by 0x48D4296: g_log_writer_format_fields (gmessages.c:2348)
==25372==    by 0x48D501D: g_log_writer_standard_streams (gmessages.c:2609)
==25372==    by 0x48D5121: g_log_writer_default (gmessages.c:2713)
==25372==    by 0x48D34D6: g_log_structured_array (gmessages.c:1970)
==25372==    by 0x48D34D6: g_log_structured_array (gmessages.c:1943)
==25372==    by 0x48D390C: g_log_default_handler (gmessages.c:3158)
==25372==    by 0x48D3B5E: g_logv (gmessages.c:1370)
==25372==    by 0x48D3D4E: g_log (gmessages.c:1432)
==25372==    by 0x55FD715: backend_init (sqlite.c:207)
==25372==    by 0x485C34E: j_backend_smd_init (jbackend.c:663)
==25372==    by 0x485DBCB: j_init (jcommon.c:191)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 4 bytes in 1 blocks are still reachable in loss record 8 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x4911CF2: g_private_impl_new (gthread-posix.c:1007)
==25372==    by 0x49122C3: g_private_get_impl (gthread-posix.c:1035)
==25372==    by 0x49122C3: g_private_set (gthread-posix.c:1082)
==25372==    by 0x48F46E7: g_thread_proxy (gthread.c:766)
==25372==    by 0x49F0181: start_thread (pthread_create.c:486)
==25372==    by 0x4B25B1E: clone (clone.S:95)
==25372== 
==25372== 15 bytes in 1 blocks are still reachable in loss record 15 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D24D0: g_malloc (gmem.c:99)
==25372==    by 0x48EB75E: g_strdup (gstrfuncs.c:363)
==25372==    by 0x48A7098: g_get_charset (gcharset.c:204)
==25372==    by 0x48D4296: g_log_writer_format_fields (gmessages.c:2348)
==25372==    by 0x48D501D: g_log_writer_standard_streams (gmessages.c:2609)
==25372==    by 0x48D5121: g_log_writer_default (gmessages.c:2713)
==25372==    by 0x48D34D6: g_log_structured_array (gmessages.c:1970)
==25372==    by 0x48D34D6: g_log_structured_array (gmessages.c:1943)
==25372==    by 0x48D390C: g_log_default_handler (gmessages.c:3158)
==25372==    by 0x48D3B5E: g_logv (gmessages.c:1370)
==25372==    by 0x48D3D4E: g_log (gmessages.c:1432)
==25372==    by 0x55FD715: backend_init (sqlite.c:207)
==25372==    by 0x485C34E: j_backend_smd_init (jbackend.c:663)
==25372==    by 0x485DBCB: j_init (jcommon.c:191)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 15 bytes in 1 blocks are still reachable in loss record 16 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D24D0: g_malloc (gmem.c:99)
==25372==    by 0x48EB75E: g_strdup (gstrfuncs.c:363)
==25372==    by 0x48A7105: g_get_charset (gcharset.c:206)
==25372==    by 0x48D4296: g_log_writer_format_fields (gmessages.c:2348)
==25372==    by 0x48D501D: g_log_writer_standard_streams (gmessages.c:2609)
==25372==    by 0x48D5121: g_log_writer_default (gmessages.c:2713)
==25372==    by 0x48D34D6: g_log_structured_array (gmessages.c:1970)
==25372==    by 0x48D34D6: g_log_structured_array (gmessages.c:1943)
==25372==    by 0x48D390C: g_log_default_handler (gmessages.c:3158)
==25372==    by 0x48D3B5E: g_logv (gmessages.c:1370)
==25372==    by 0x48D3D4E: g_log (gmessages.c:1432)
==25372==    by 0x55FD715: backend_init (sqlite.c:207)
==25372==    by 0x485C34E: j_backend_smd_init (jbackend.c:663)
==25372==    by 0x485DBCB: j_init (jcommon.c:191)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 16 bytes in 1 blocks are still reachable in loss record 99 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D2518: g_malloc0 (gmem.c:129)
==25372==    by 0x48A75D1: g_get_language_names (gcharset.c:566)
==25372==    by 0x48C264B: g_key_file_init (gkeyfile.c:637)
==25372==    by 0x48C44BC: g_key_file_new (gkeyfile.c:694)
==25372==    by 0x485DF35: j_configuration_new (jconfiguration.c:167)
==25372==    by 0x485D99B: j_init (jcommon.c:152)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 24 bytes in 1 blocks are still reachable in loss record 108 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D2518: g_malloc0 (gmem.c:129)
==25372==    by 0x48A7129: g_get_charset (gcharset.c:190)
==25372==    by 0x48D4296: g_log_writer_format_fields (gmessages.c:2348)
==25372==    by 0x48D501D: g_log_writer_standard_streams (gmessages.c:2609)
==25372==    by 0x48D5121: g_log_writer_default (gmessages.c:2713)
==25372==    by 0x48D34D6: g_log_structured_array (gmessages.c:1970)
==25372==    by 0x48D34D6: g_log_structured_array (gmessages.c:1943)
==25372==    by 0x48D390C: g_log_default_handler (gmessages.c:3158)
==25372==    by 0x48D3B5E: g_logv (gmessages.c:1370)
==25372==    by 0x48D3D4E: g_log (gmessages.c:1432)
==25372==    by 0x55FD715: backend_init (sqlite.c:207)
==25372==    by 0x485C34E: j_backend_smd_init (jbackend.c:663)
==25372==    by 0x485DBCB: j_init (jcommon.c:191)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 40 bytes in 1 blocks are still reachable in loss record 136 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x4911E4F: g_rec_mutex_impl_new (gthread-posix.c:276)
==25372==    by 0x4911F47: g_rec_mutex_get_impl (gthread-posix.c:302)
==25372==    by 0x4911F47: g_rec_mutex_lock (gthread-posix.c:387)
==25372==    by 0x508C732: g_module_open (gmodule.c:500)
==25372==    by 0x485A46D: j_backend_load (jbackend.c:70)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DA8B: j_init (jcommon.c:171)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 40 bytes in 1 blocks are still reachable in loss record 137 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D24D0: g_malloc (gmem.c:99)
==25372==    by 0x508CC47: g_module_open (gmodule.c:631)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DA8B: j_init (jcommon.c:171)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 40 bytes in 1 blocks are still reachable in loss record 138 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D24D0: g_malloc (gmem.c:99)
==25372==    by 0x508CC47: g_module_open (gmodule.c:631)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DB17: j_init (jcommon.c:180)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 40 bytes in 1 blocks are still reachable in loss record 139 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D24D0: g_malloc (gmem.c:99)
==25372==    by 0x508CC47: g_module_open (gmodule.c:631)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DBA3: j_init (jcommon.c:189)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 61 bytes in 1 blocks are still reachable in loss record 141 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x401D0E9: strdup (strdup.c:42)
==25372==    by 0x40098BA: _dl_map_object (dl-load.c:2398)
==25372==    by 0x40149A3: dl_open_worker (dl-open.c:228)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DB17: j_init (jcommon.c:180)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 61 bytes in 1 blocks are still reachable in loss record 142 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x400C6CF: _dl_new_object (dl-object.c:163)
==25372==    by 0x4006487: _dl_map_object_from_fd (dl-load.c:1001)
==25372==    by 0x4009932: _dl_map_object (dl-load.c:2466)
==25372==    by 0x40149A3: dl_open_worker (dl-open.c:228)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DB17: j_init (jcommon.c:180)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 61 bytes in 1 blocks are still reachable in loss record 143 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D24D0: g_malloc (gmem.c:99)
==25372==    by 0x48EB75E: g_strdup (gstrfuncs.c:363)
==25372==    by 0x508CC52: g_module_open (gmodule.c:632)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DB17: j_init (jcommon.c:180)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 62 bytes in 1 blocks are still reachable in loss record 144 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x401D0E9: strdup (strdup.c:42)
==25372==    by 0x40098BA: _dl_map_object (dl-load.c:2398)
==25372==    by 0x40149A3: dl_open_worker (dl-open.c:228)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DBA3: j_init (jcommon.c:189)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 62 bytes in 1 blocks are still reachable in loss record 145 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x400C6CF: _dl_new_object (dl-object.c:163)
==25372==    by 0x4006487: _dl_map_object_from_fd (dl-load.c:1001)
==25372==    by 0x4009932: _dl_map_object (dl-load.c:2466)
==25372==    by 0x40149A3: dl_open_worker (dl-open.c:228)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DBA3: j_init (jcommon.c:189)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 62 bytes in 1 blocks are still reachable in loss record 146 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D24D0: g_malloc (gmem.c:99)
==25372==    by 0x48EB75E: g_strdup (gstrfuncs.c:363)
==25372==    by 0x508CC52: g_module_open (gmodule.c:632)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DBA3: j_init (jcommon.c:189)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 64 bytes in 1 blocks are still reachable in loss record 150 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x401D0E9: strdup (strdup.c:42)
==25372==    by 0x40098BA: _dl_map_object (dl-load.c:2398)
==25372==    by 0x40149A3: dl_open_worker (dl-open.c:228)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DA8B: j_init (jcommon.c:171)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 64 bytes in 1 blocks are still reachable in loss record 151 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x400C6CF: _dl_new_object (dl-object.c:163)
==25372==    by 0x4006487: _dl_map_object_from_fd (dl-load.c:1001)
==25372==    by 0x4009932: _dl_map_object (dl-load.c:2466)
==25372==    by 0x40149A3: dl_open_worker (dl-open.c:228)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DA8B: j_init (jcommon.c:171)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 64 bytes in 1 blocks are still reachable in loss record 152 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D24D0: g_malloc (gmem.c:99)
==25372==    by 0x48EB75E: g_strdup (gstrfuncs.c:363)
==25372==    by 0x508CC52: g_module_open (gmodule.c:632)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DA8B: j_init (jcommon.c:171)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 64 bytes in 1 blocks are still reachable in loss record 153 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D24D0: g_malloc (gmem.c:99)
==25372==    by 0x489FC22: g_async_queue_new_full (gasyncqueue.c:141)
==25372==    by 0x48F4F9C: g_thread_pool_new (gthreadpool.c:494)
==25372==    by 0x485C7A9: j_background_operation_init (jbackground-operation.c:137)
==25372==    by 0x485DC1F: j_init (jcommon.c:200)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 88 bytes in 1 blocks are still reachable in loss record 191 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D24D0: g_malloc (gmem.c:99)
==25372==    by 0x48E9B12: g_slice_alloc (gslice.c:1025)
==25372==    by 0x48BB05D: g_hash_table_new_full (ghash.c:717)
==25372==    by 0x48DCACA: g_quark_init (gquark.c:61)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 96 bytes in 1 blocks are still reachable in loss record 240 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x40120AF: _dl_check_map_versions (dl-version.c:274)
==25372==    by 0x4014CFB: dl_open_worker (dl-open.c:277)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DBA3: j_init (jcommon.c:189)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 120 bytes in 1 blocks are still reachable in loss record 246 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x40120AF: _dl_check_map_versions (dl-version.c:274)
==25372==    by 0x4014CFB: dl_open_worker (dl-open.c:277)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DA8B: j_init (jcommon.c:171)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 136 bytes in 1 blocks are still reachable in loss record 251 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x4007DB1: open_path (dl-load.c:2112)
==25372==    by 0x4009AB7: _dl_map_object (dl-load.c:2297)
==25372==    by 0x400E5A1: openaux (dl-deps.c:64)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x400E925: _dl_map_object_deps (dl-deps.c:248)
==25372==    by 0x4014A0A: dl_open_worker (dl-open.c:271)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DB17: j_init (jcommon.c:180)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372== 
==25372== 136 bytes in 1 blocks are still reachable in loss record 252 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x400C6CF: _dl_new_object (dl-object.c:163)
==25372==    by 0x4006487: _dl_map_object_from_fd (dl-load.c:1001)
==25372==    by 0x4009932: _dl_map_object (dl-load.c:2466)
==25372==    by 0x400E5A1: openaux (dl-deps.c:64)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x400E925: _dl_map_object_deps (dl-deps.c:248)
==25372==    by 0x4014A0A: dl_open_worker (dl-open.c:271)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DB17: j_init (jcommon.c:180)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372== 
==25372== 256 bytes in 1 blocks are still reachable in loss record 259 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D2518: g_malloc0 (gmem.c:129)
==25372==    by 0x48BA806: g_hash_table_resize (ghash.c:594)
==25372==    by 0x48BAA43: g_hash_table_maybe_resize (ghash.c:649)
==25372==    by 0x48BAA43: g_hash_table_insert_node (ghash.c:984)
==25372==    by 0x48BADEC: g_hash_table_insert_internal (ghash.c:1235)
==25372==    by 0x48A7535: read_aliases (gcharset.c:281)
==25372==    by 0x48A7535: unalias_lang (gcharset.c:300)
==25372==    by 0x48A7535: g_get_language_names (gcharset.c:587)
==25372==    by 0x48C264B: g_key_file_init (gkeyfile.c:637)
==25372==    by 0x48C44BC: g_key_file_new (gkeyfile.c:694)
==25372==    by 0x485DF35: j_configuration_new (jconfiguration.c:167)
==25372==    by 0x485D99B: j_init (jcommon.c:152)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 336 bytes in 2 blocks are still reachable in loss record 260 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x40120AF: _dl_check_map_versions (dl-version.c:274)
==25372==    by 0x4014CFB: dl_open_worker (dl-open.c:277)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DB17: j_init (jcommon.c:180)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 512 bytes in 1 blocks are still reachable in loss record 266 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D2518: g_malloc0 (gmem.c:129)
==25372==    by 0x48BA7D7: g_hash_table_resize (ghash.c:589)
==25372==    by 0x48BAA43: g_hash_table_maybe_resize (ghash.c:649)
==25372==    by 0x48BAA43: g_hash_table_insert_node (ghash.c:984)
==25372==    by 0x48BADEC: g_hash_table_insert_internal (ghash.c:1235)
==25372==    by 0x48A7535: read_aliases (gcharset.c:281)
==25372==    by 0x48A7535: unalias_lang (gcharset.c:300)
==25372==    by 0x48A7535: g_get_language_names (gcharset.c:587)
==25372==    by 0x48C264B: g_key_file_init (gkeyfile.c:637)
==25372==    by 0x48C44BC: g_key_file_new (gkeyfile.c:694)
==25372==    by 0x485DF35: j_configuration_new (jconfiguration.c:167)
==25372==    by 0x485D99B: j_init (jcommon.c:152)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 512 bytes in 1 blocks are still reachable in loss record 267 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D2518: g_malloc0 (gmem.c:129)
==25372==    by 0x48BA7F5: g_hash_table_resize (ghash.c:593)
==25372==    by 0x48BAA43: g_hash_table_maybe_resize (ghash.c:649)
==25372==    by 0x48BAA43: g_hash_table_insert_node (ghash.c:984)
==25372==    by 0x48BADEC: g_hash_table_insert_internal (ghash.c:1235)
==25372==    by 0x48A7535: read_aliases (gcharset.c:281)
==25372==    by 0x48A7535: unalias_lang (gcharset.c:300)
==25372==    by 0x48A7535: g_get_language_names (gcharset.c:587)
==25372==    by 0x48C264B: g_key_file_init (gkeyfile.c:637)
==25372==    by 0x48C44BC: g_key_file_new (gkeyfile.c:694)
==25372==    by 0x485DF35: j_configuration_new (jconfiguration.c:167)
==25372==    by 0x485D99B: j_init (jcommon.c:152)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 1,192 bytes in 1 blocks are still reachable in loss record 269 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x400C3ED: _dl_new_object (dl-object.c:73)
==25372==    by 0x4006487: _dl_map_object_from_fd (dl-load.c:1001)
==25372==    by 0x4009932: _dl_map_object (dl-load.c:2466)
==25372==    by 0x400E5A1: openaux (dl-deps.c:64)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x400E925: _dl_map_object_deps (dl-deps.c:248)
==25372==    by 0x4014A0A: dl_open_worker (dl-open.c:271)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DB17: j_init (jcommon.c:180)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372== 
==25372== 1,237 bytes in 1 blocks are still reachable in loss record 270 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x400C3ED: _dl_new_object (dl-object.c:73)
==25372==    by 0x4006487: _dl_map_object_from_fd (dl-load.c:1001)
==25372==    by 0x4009932: _dl_map_object (dl-load.c:2466)
==25372==    by 0x40149A3: dl_open_worker (dl-open.c:228)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DB17: j_init (jcommon.c:180)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 1,238 bytes in 1 blocks are still reachable in loss record 271 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x400C3ED: _dl_new_object (dl-object.c:73)
==25372==    by 0x4006487: _dl_map_object_from_fd (dl-load.c:1001)
==25372==    by 0x4009932: _dl_map_object (dl-load.c:2466)
==25372==    by 0x40149A3: dl_open_worker (dl-open.c:228)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DBA3: j_init (jcommon.c:189)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 1,240 bytes in 1 blocks are still reachable in loss record 272 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x400C3ED: _dl_new_object (dl-object.c:73)
==25372==    by 0x4006487: _dl_map_object_from_fd (dl-load.c:1001)
==25372==    by 0x4009932: _dl_map_object (dl-load.c:2466)
==25372==    by 0x40149A3: dl_open_worker (dl-open.c:228)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x40142A5: _dl_open (dl-open.c:599)
==25372==    by 0x50D7255: dlopen_doit (dlopen.c:66)
==25372==    by 0x4B665CE: _dl_catch_exception (dl-error-skeleton.c:196)
==25372==    by 0x4B6665E: _dl_catch_error (dl-error-skeleton.c:215)
==25372==    by 0x50D7A24: _dlerror_run (dlerror.c:163)
==25372==    by 0x50D72E5: dlopen@@GLIBC_2.2.5 (dlopen.c:87)
==25372==    by 0x508C9F1: _g_module_open (gmodule-dl.c:98)
==25372==    by 0x508C9F1: g_module_open (gmodule.c:599)
==25372==    by 0x485A4D9: j_backend_load (jbackend.c:79)
==25372==    by 0x485A90A: j_backend_load_client (jbackend.c:183)
==25372==    by 0x485DA8B: j_init (jcommon.c:171)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 2,032 bytes in 1 blocks are still reachable in loss record 273 of 275
==25372==    at 0x483CB35: calloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D2518: g_malloc0 (gmem.c:129)
==25372==    by 0x48E9D53: thread_memory_from_self (gslice.c:519)
==25372==    by 0x48E9D53: thread_memory_from_self (gslice.c:505)
==25372==    by 0x48E9D53: g_slice_alloc (gslice.c:1003)
==25372==    by 0x48BB05D: g_hash_table_new_full (ghash.c:717)
==25372==    by 0x48DCACA: g_quark_init (gquark.c:61)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== 16,384 bytes in 1 blocks are still reachable in loss record 275 of 275
==25372==    at 0x483A74F: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==25372==    by 0x48D24D0: g_malloc (gmem.c:99)
==25372==    by 0x48DCADB: g_quark_init (gquark.c:62)
==25372==    by 0x4010959: call_init.part.0 (dl-init.c:72)
==25372==    by 0x4010A58: call_init (dl-init.c:30)
==25372==    by 0x4010A58: _dl_init (dl-init.c:119)
==25372==    by 0x40010C9: ??? (in /usr/lib/x86_64-linux-gnu/ld-2.29.so)
==25372== 
==25372== LEAK SUMMARY:
==25372==    definitely lost: 0 bytes in 0 blocks
==25372==    indirectly lost: 0 bytes in 0 blocks
==25372==      possibly lost: 0 bytes in 0 blocks
==25372==    still reachable: 26,394 bytes in 40 blocks
==25372==                       of which reachable via heuristic:
==25372==                         newarray           : 1,536 bytes in 16 blocks
==25372==         suppressed: 19,682 bytes in 327 blocks
==25372== 
==25372== For counts of detected and suppressed errors, rerun with: -v
==25372== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 18 from 18)
